playground/mocks/data/idp/idx/identify.json

```diff
@@ -18,7 +18,8 @@
             "value":[
               {
                   "name":"identifier",
-                  "label":"Username"
+                  "label":"Username",
+                  "required": true
```

Comment:
Pretty sure this was a bad mock since identifier is a required field to proceed. Not sure why it's like this, maybe this was the actual API response way in the past and v2 widget has hardcoded logic to say it's actually required. v3 totally trusts the API response, so we need to have the "required" field set as true otherwise it will allow the user to click Submit without this field filled.